lib/plugins/chat.js

```diff
@@ -171,7 +172,9 @@ function inject (bot, options) {
   bot.tabComplete = callbackify(tabComplete)
 
   function addDefaultPatterns () {
-    const USERNAME_REGEX = '(?:\\(.+\\)|\\[.+\\]|.)*?(\\w+)'
+    if (!defaultChatPatterns) return
+    // original regex: (?:\\(.+\\)|\\[.+\\]|.)*?(\\w+)
```

Comment:
why leave the original regex

Reply:
In case someone wanted to see what the original regex looked like before I changed it

Through git blame exists

Reply:
please remove it

Reply:
however that seems unrelated with this PR, why did you make that change?

Reply:
I made that change since I believed it would be a vulnerability if anyone could just chat a series of `[]` or `()` and crash any mineflayer bot.

Reply:
I'll revert my regex change.